src/Linter/Sniff/Visitor/ConfigNoCacheVisitor.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace Helmich\TypoScriptLint\Linter\Sniff\Visitor;
+
+use Helmich\TypoScriptLint\Linter\Report\Issue;
+use Helmich\TypoScriptLint\Linter\Sniff\ConfigNoCacheSniff;
+use Helmich\TypoScriptParser\Parser\AST\Operator\Assignment;
+use Helmich\TypoScriptParser\Parser\AST\Statement;
+
+class ConfigNoCacheVisitor implements SniffVisitor
+{
+    /** @var Issue[] */
+    private $issues = [];
+
+    /**
+     * @return Issue[]
+     */
+    public function getIssues(): array
+    {
+        return $this->issues;
+    }
+
+    public function enterTree(array $statements): void
+    {
+    }
+
+    public function enterNode(Statement $statement): void
+    {
+         if (!$statement instanceof Assignment) {
+             return;
+         }
+         if ($statement->object->relativeName !== 'no_cache'
+             && !str_ends_with($statement->object->relativeName, '.no_cache')) {
+             return;
+         }
+         if ($statement->value->value !== '0') {
+             $this->issues[] = new Issue(
+                $statement->sourceLine,
+                null,
+                sprintf(
+                    'Setting config.no_cache = 1 is discouraged as it is bad for performance.'
```

Comment:
Nitpick: There'll be a space character missing in the concatenated issue message.

```suggestion
                    'Setting config.no_cache = 1 is discouraged as it is bad for performance. '
```